core/lib/spree/app_configuration.rb

```diff
@@ -415,6 +415,14 @@ def default_pricing_options
     #   Spree::OrderContents.
     class_name_attribute :order_contents_class, default: 'Spree::OrderContents'
 
+    # Allows providing your own class for prioritizing store credit application
+    # to an order.
+    #
+    # @!attribute [rw] order_credit_sorter_class
+    # @return [Class] a class with the same public interfaces as
+    #   Spree::OrderCreditSorter.
+    class_name_attribute :order_credit_sorter_class, default: 'Spree::OrderCreditSorter'
```

Comment:
What about `:store_credit_sorter_class` & `Spree::StoreCreditSorter` as the names? I'm not very familiar with that sub-system, so reading "order credit" made me think that maybe it was something different than store credits.

Reply:
I'm fine with that name if we prefer - my only possible reservation about it is that there could be multiple contexts in which you'd want to "sort store credits". For example, maybe the UI for rendering the list of a users credits. That could be slightly confusing, potentially.

What do you think, @waiting-for-dev ? I'm more than happy to change the name if you think that makes the purpose more obvious!

Reply:
That's a good point. What about something along the lines of `Spree::StoreCreditUtilizationSorter`? (feel free to find a better name 🙂 )

Reply:
Renamed to `Spree::StoreCreditPrioritizer` - thoughts?